packages/shared/hatch-sleep-types.ts

```diff
@@ -6,6 +6,7 @@ export const Product = {
   restMini: 'restMini',
   restore: 'restore',
   restoreIot: 'restoreIot',
+  restoreV4 = 'restoreV4',
```

Comment:
I converted this an object instead of an enum so the syntax is slightly different now

```suggestion
  restoreV4: 'restoreV4',
```